<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CMDMITTRN9" name="Default test" tests="7" failures="1" timestamp="8 Dec 2016 11:17:18 GMT" time="24.902" errors="0">
  <testcase name="GL_Settings_AE_09" time="1.766" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
  <testcase name="GL_Settings_AE_10" time="5.579" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
  <testcase name="GL_Settings_AE_11" time="2.572" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
  <testcase name="GL_Settings_AE_12" time="1.38" classname="com.glance.dashboard.tests.VerifyAddEngagement">
    <failure type="java.lang.AssertionError" message="expected [[Ljava.lang.String;@73eb8dd7] but found [Dashboard Management]">
      <![CDATA[java.lang.AssertionError: expected [[Ljava.lang.String;@73eb8dd7] but found [Dashboard Management]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertArrayEquals(Assert.java:169)
	at org.testng.Assert.assertEquals(Assert.java:113)
	at org.testng.Assert.assertEquals(Assert.java:179)
	at com.glance.dashboard.tests.VerifyAddEngagement.GL_Settings_AE_12(VerifyAddEngagement.java:145)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- GL_Settings_AE_12 -->
  <testcase name="GL_Settings_AE_11_verify1" time="1.814" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
  <testcase name="GL_Settings_AE_11_verify2" time="2.077" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
  <testcase name="GL_Settings_AE_13" time="0.785" classname="com.glance.dashboard.tests.VerifyAddEngagement"/>
</testsuite> <!-- Default test -->
